doc/try_new_features_before_release.md

```diff
@@ -52,3 +52,24 @@ python -m venv env
 # Activate it
 env\Scripts\activate.ps1
 ```
+
+## Install from source code
```

Comment:
remind me that https://github.com/Azure/azure-cli/blob/dev/doc/configuring_your_machine.md need to be updated.

Reply:
We need a documentation engineer.